src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -138,6 +138,8 @@ def create_resource(self, name, **kwargs):
             template = 'az storage account create -n {} -g {} -l {} --sku {} --kind {} --https-only'
             if not self.allow_blob_public_access:
                 template += ' --allow-blob-public-access false'
+            else:
+                template += ' --allow-blob-public-access true'
```

Comment:
Since service default changed, need to manually specify --allow-blob-public-access true